docs/pages/API/View/index.js

```diff
@@ -0,0 +1,387 @@
+import React, { PureComponent } from 'react'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { duotoneEarth } from 'react-syntax-highlighter/dist/styles/prism'
+import { Text } from '../../../../src'
+import ViewBasicSpecimen from '../../../specimens/ViewBasic'
+import style from '../style.less'
+
+const apiOptions = [{
+   name: 'className',
+   type: 'String',
+   default: '-',
+   description: 'Pass a <code>className</code> to customize className of the OIO Container'
+}, {
+   name: 'display',
+   type: 'String',
+   default: '<code>block</code>',
+   description: 'Any css block options can be used including: <code>block</code>, <code>flex</code>, <code>inline-block</code>'
+}, {
+   name: 'float',
+   type: 'String',
+   default: '-',
+   description: 'Any css float options can be used including: <code>left</code>, <code>right</code>'
+}, {
+   name: 'position',
+   type: 'String',
+   default: '<code>relative</code>',
+   description: 'Any css position options can be used including: <code>relative</code>, <code>absolute</code>, <code>fixed</code>'
+}, {
+   name: 'top',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'left',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'right',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'bottom',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'height',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'maxHeight',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'maxWidth',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'minHeight',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'minWidth',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'width',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}, {
+   name: 'style',
+   type: 'Object',
+   default: '<code>{}</code>',
+   description: 'Pass a style <code>object</code> to customize inline style of the OIO Container'
+}, {
+   name: 'zIndex',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value'
+}]
+
+const appearanceApiOptions = [{
+   name: 'backgroundColor',
+   type: 'String',
+   default: '-',
+   description: 'Use any valid css color'
+}, {
+   name: 'backgroundImage',
+   type: 'String',
+   default: '-',
+   description: 'specifiy a background imagee'
```

Comment:
Typo

Reply:
Also not a big deal, but watch out for the mixed case for starting word of description